test/integration/test-submit-custom.js

```diff
@@ -80,15 +71,11 @@ server.listen(common.port, function() {
     path: '/'
   }, function(err, res) {
 
-    if (err) {
-      throw err;
-    }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'throw'.](https://www.codacy.com/app/form-data/form-data/file/1700090167/issues/source?bid=2805153&fileBranchId=3045625#l74)